core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/impl/ContentUpgradeImpl.java

```diff
@@ -227,6 +225,12 @@ public ContentUpgrade doSetProperty(@Nonnull String name, Object value) {
         return this;
     }
 
+    @Override
+    public ContentUpgrade doForceProperty(@Nonnull String name, Object value) {
+        actions.add(new SetProperty(name, value));
```

Comment:
I don't see any difference to doSetProperty(...). Maybe you meant to call new ForceProperty?